### ISSUE TITLE
cross fertilization with ABES / SUDOC

### ISSUE THREAD

**Issue by maxlath** [inventaire/inventaire#70]
[moving @AmauryVanEspen's message originally posted in #31 thread in a dedicated issue]
"Hi,

what would you think about cross fertilization with ABES / SUDOC ? (like World Cat)
http://www.slideshare.net/jsicot/utiliser-les-apiwebservices
http://www.sudoc.abes.fr/
http://www.abes.fr/

Best

Amaury"

**Comment by maxlath** [comment]
@AmauryVanEspen yes! I'm currently working on [restructuring the book database](https://trello.com/c/doDQZ987/99-restructure-de-database-to-allow-more-detailed-data) to act as a Wikidata books data vanguard, which implies that data produced in inventaire will be released with CC0 license (as required to be integrated to Wikidata), letting other data sources copying it freely. So that cross fertilization could definitely happen in this direction, but what about the reverse way? Do you know which license ABES / SUDOC use?

**Comment by AmauryVanEspen** [comment]
Dear Maxime,

I don't know yet, but these is a meeting in few days about this subject.
I have added Thomas broussaud, from Lille, who will take part in the meeting
I could be such a good idea to cross the data
have you ever heard about Communecter.org also ? and the CommonsDev team on
GitHub ?

Best

Amaury

**Comment by maxlath** [comment]
Great! I've been in touch with Yann Nicolas from ABES a few months ago, but I would be glad to resume the discussion on integrating inventaire and ABES / SUDOC. I was primarily thinking of how to show resources availability in surrounding university libraries but cross fertilization could definitely be interesting! It's still a bit early for inventaire though, as I'm just at the middle of restructuring the data system: hopefully, this will all makes more sense in a few months

I'm aware of communecter.org's work but we haven't discussed much yet.
And regarding CommonsDev, yes, I did met @almereyda and @SimonSarazin a few times already ^^

**Comment by AmauryVanEspen** [comment]
Dear Maxime,
CC : Tibor (Communecter), Maia (Semeoz), Simon (EnCommuns)

You should open a talk, maybe in few weeks, regarding to your plan, with
Tibor and Maïa, in copy of this E-Mail.
I would be this afternoon with president of French University Communities
about the subject of "open data" about availibity of "books", medias and
ressources to share (especially metadata and API availibity)

In my personnal goal, i would like, first of all, be able to get a list of
institution (particulary "Research Laboratory", and their members), via
"geocode" or at least, address, and contact information, in order to feed
an open database ...
That, in my opinion, the first step (in "My" project) before to get
metadata about medias and books/ressources availaibity.
My goal is about "physical crossfertilization" and "mapping the Mixer"
(related to the Mixer Theory ... about social network interaction and
ability to innovate through creativity and knowlegde economy with inner
"business cluster and ecosystem")

So, i will let you know about it, if you are interested
let me know, how i could be useful to your project and the opportunity to
add fields about "organisations" (more than individuals) which store
medias/books/ressources ...
I would be such interesting to have an entity "Organisation" (linked to
individuals) in inventaire.io ... (and different from "team")
My wish would be to use the right ontology and taxonomy of metadata to feed
"localisation" informations about the organisation.

Simon has already worked on a similar project about "research project"
through encommuns.org similar platform or imagination.social
I do not remember the platform they make ... may he will be able to answer

Best

Amaury

**Comment by maxlath** [comment]
hello @AmauryVanEspen, I'm looking at opportunities to use other databases identifiers to de-duplicate works: do you know if the SUDOC has identifiers for works that could be found from their edition's ISBNs?
